<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Cell_Main</class>
 <widget class="QMainWindow" name="Cell_Main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1165</width>
    <height>816</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QT图书管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QStackedWidget{
	background-color:#c8aeff;
}
QPushButton[name=&quot;btn&quot;]{
	border-radius:4px;
	background:#9c9ce8;
	color:black;
	font:16px;
	min-height:28px;
	min-width:85px;
	padding:4px;
}
QPushButton[name=&quot;btn&quot;]:pressed{
	border-radius:4px;
	background:#7070ff;
	color:black;
	font:16px;
	min-height:28px;
	min-width:85px;
	padding:4px;
}
QPushButton[name=&quot;btnr&quot;]{
	border-radius:4px;
	background:#ff0000;
	color:yellow;
	font:16px;
	min-height:28px;
	min-width:85px;
	padding:4px;
}
QLineEdit[name=&quot;le&quot;]{
	border-radius:4px;
	min-height:25px;
	border:1px solid black;
	color:black;
}
QLabel[name=&quot;title&quot;]{
	font:18px '黑体‘;
	color:black;
}
QLabel[name=&quot;lb&quot;]{
	font:20px '楷体‘;
	color:black;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#tool{
	background: #ababff;
}


QWidget#tool QToolButton {
    background: #ababff;
    color: black;
    min-height: 55px;
    font: 14px '黑体';
    qproperty-iconSize: 18px 18px;
}


QWidget#tool QToolButton:checked {
    background: #2f2f46;
}

</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <property name="spacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QWidget" name="tool" native="true">
      <property name="minimumSize">
       <size>
        <width>120</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>120</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
       <item row="1" column="0">
        <widget class="QToolButton" name="btn_book">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>  图书管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/Reading.png</normaloff>:/Reading.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QToolButton" name="btn_user">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>  用户管理</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/SetUp.png</normaloff>:/SetUp.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
       <item row="3" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QToolButton" name="btn_his">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <family>黑体</family>
           <pointsize>-1</pointsize>
           <italic>false</italic>
           <bold>false</bold>
           <kerning>false</kerning>
          </font>
         </property>
         <property name="text">
          <string>  借阅记录</string>
         </property>
         <property name="icon">
          <iconset resource="../res/res.qrc">
           <normaloff>:/Files.png</normaloff>:/Files.png</iconset>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextBesideIcon</enum>
         </property>
         <attribute name="buttonGroup">
          <string notr="true">buttonGroup</string>
         </attribute>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
